S/SPIRV_LLVM_Backend/build_tarballs.jl

```diff
@@ -0,0 +1,93 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SPIRV_LLVM_Backend"
+version = v"19"
```

Comment:
Only loosely following LLVM's versioning here so that we can still make minor releases ourselves (e.g. containing Julia-specific patches).